src/plots/polar/polar.js

```diff
@@ -333,9 +335,11 @@ proto.mockCartesianAxis = function(fullLayout, polarLayout, opts) {
 
     ax.setRange = function() {
         var sectorBBox = _this.sectorBBox;
-        var rl = _this.radialAxis._rl;
-        var drl = rl[1] - rl[0];
         var ind = bboxIndices[axId];
+        var rl = _this.radialAxis._rl;
+        var radius = _this.radius;
+        var innerRadius = _this.innerRadius;
+        var drl = radius * (rl[1] - rl[0]) / (radius - innerRadius);
```

Comment:
you don't have access to `hole` here to just do `(rl[1] - rl[0]) / (1 - hole)`?

Reply:
Yep, the `polayLayout` is passed to the `proto.mockCartesianAxis`, so yeah this line could be:

```js
var drl = (rl[1] - rl[0]) / (1 - polarLayout.hole)
```

if you prefer.

Reply:
nonblocking but that would be marginally simpler.

Reply:
Simpler is better if it's _marginally_ simpler -> https://github.com/plotly/plotly.js/pull/2977/commits/35199c638fb00e939b7b5e38b99a4207dbd9ee21